src/views/Home/components/MetricsSection/index.tsx

```diff
@@ -8,9 +8,9 @@ import IconCard, { IconCardData } from '../IconCard'
 import StatCardContent from './StatCardContent'
 import GradientLogo from '../GradientLogoSvg'
 
-// Values fetched from bitQuery effective 28/7/21
-const txCount = 30673865
-const addressCount = 1966700
+// Values fetched from bitQuery effective 5/8/21
+const txCount = 37553080
+const addressCount = 2279782
```

Comment:
Sorry if I missed this, but why are these hard-coded in here?

Reply:
@hachiojidev - we're waiting on Debank to add these to their API, I approved this being hardcoded until they add this

Reply:
We're waiting on Debank providing this info as part of their api response (that we currently use for TVL). So until then it's hard coded data coming from a bitquery response.